anaconda.py

```diff
@@ -96,8 +96,7 @@ def exitHandler(rebootData, storage):
 
     anaconda.dbus_launcher.stop()
 
-    if not flags.imageInstall and not flags.livecdInstall \
-       and not flags.dirInstall:
+    if not flags.imageInstall and not flags.dirInstall and conf.system.can_reboot:
```

Comment:
shouldn't this be just
if conf.system.can_reboot

Reply:
Ok, let's do that.

Reply:
Fixed.